### DISCUSSION TITLE
GitHub Classroom Autograding not working

### DISCUSSION THREAD

**Discussion by ghost** [community/community#50445] [GitHub Education]
Hello. Something is wrong with my github classroom. I have successfully written my assignment and in my codespace the tests appear as completed. But in my assignment repository when i click on "pull requests" it says this - "GitHub Classroom Workflow / Autograding (push) Failing after 4s — Autograding". When i click on "details" there is a red x by "Run education/autograding@v1" and this is what it shows:
Run education/autograding@v1

::stop-commands::***

📝 1

Invalid input type.

❌ 1
::error::The output for test 1 did not match%0AExpected:%0A3%0A1 4%0A0 1%0A1 3%0AActual:%0AInvalid input type.
📝 2

Invalid input type.

❌ 2
::error::The output for test 2 did not match%0AExpected:%0A0%0AActual:%0AInvalid input type.
📝 4

Invalid input type.

❌ 4
::error::The output for test 4 did not match%0AExpected:%0A99990%0AActual:%0AInvalid input type.

::***::
Points 0/3
Warning: The set-output command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Comment by kvk-code** [comment]
Please look into these guides that I have prepared: 


(1)How to Submit basic Git and GitHub Learning Resources Assignments through pull requests (Link: https://youtu.be/IRuFEMVNTL4  )

(2)How to Submit a Sample Auto-grading GitHub Classroom Coding Assignment using GitHub Codespaces
(Link: https://youtu.be/5o3PalpZd5U  )

**Reply by colematt** [reply]
I had a similar problem to @Daniels-221RDB373 but unfortunately neither of these videos were helpful. They discuss the process from the student's viewpoint, showing how they would submit their work if the repository were working. But in this case, our repository's autograder is _not_ working properly. For any teachers with a similar problem, don't bother to watch the videos.
